src/library-authoring/common/index.scss

```diff
@@ -52,3 +52,7 @@
     font-weight: bold;
   }
 }
+
+.pgn__form-text-invalid {
+  color: #C32D3A;
+}
```

Comment:
curious about this one, did switching away from the depreciated components make it so we no longer have the invalid text color?

Reply:
i think there was some styling inconsistency but couldn't find its use now, removing it